extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryFrameProcessor.java

```diff
@@ -56,8 +58,10 @@
 public class WindowOperatorQueryFrameProcessor implements FrameProcessor<Object>
 {
   private static final Logger log = new Logger(WindowOperatorQueryFrameProcessor.class);
+  private final WindowOperatorQuery query;
 
   private final List<OperatorFactory> operatorFactoryList;
+  private final ObjectMapper jsonMapper;
```

Comment:
why add these back? they are unused

Reply:
I added them back because of https://github.com/apache/druid/pull/16854, to avoid unnecessary merge conflicts. I would have to add them back in the other PR otherwise.

Reply:
please remove it and add back later if needed ; I think the approach in that PR should be altered...

Reply:
>  I think the approach in that PR should be altered...so you may not need those fields

We would still need those fields, even if we do the refactoring as suggested on the other PR.

Reply:
no I don't think we will not need them there; but if we do then we can add this commit to that PR or not?

Reply:
Sure, have reverted the commit. Will add these back in the other PR when this one's merged.